CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ if(NOT CMAKE_BUILD_TYPE)
 	set(CMAKE_BUILD_TYPE "${default_build_type}" CACHE STRING "Choose the type of build." FORCE)
 endif()
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
We should also update our `README.md` file as part of this PR because it lists only C++14 (see [here](https://github.com/avast/retdec-idaplugin#requirements)):
> Requirements: 
> * A compiler supporting C++14
>     * On Windows, only Microsoft Visual C++ is supported (version >= Visual Studio 2015).

Reply:
Okay,  I will update the `README.md` as well. Thank you.

Reply:
When it comes to Windows and Microsoft Visual C++, I am not sure whether the c++17 is supported in VS 2015. It surely is supported in MS VS 2017.
There are some features in VS 2015 supported, but I am not sure whether the list in MS docs is complete.
https://docs.microsoft.com/en-us/previous-versions/hh567368(v=vs.140)?redirectedfrom=MSDN
https://devblogs.microsoft.com/cppblog/c111417-features-in-vs-2015-rtm/
https://devblogs.microsoft.com/cppblog/stl-fixes-in-vs-2015-update-3/

source:
https://stackoverflow.com/questions/41308933/how-to-enable-c17-compiling-in-visual-studio

Reply:
We should require at least Visual Studio 2017, similarly, as for RetDec.